content/applications/websites/livechat/responses.rst

```diff
@@ -1,48 +1,177 @@
+
+.. image:: responses/helpdesk-search.png
+   :align: center
+   :alt: View of the results from a helpdesk search in a Live Chat conversation.
+
+History
+-------
+
+If an operator types `/history` in the chat window, it will generate a list of the most recent pages
+the visitor has viewed on the website (up to 15).
 
-Send canned responses
-=====================
+.. image:: responses/responses-history.png
+   :align: center
+   :alt: View of the results from a /history command in a Live Chat conversation.
+
+Lead
+----
 
-| Canned responses allow you to have a full piece of text being placed when you type a shortcut word.
-  To create them, go to :menuselection:`Live Chat --> Configuration --> Canned Responses`.
-| To use them during a conversation, simply type **:** followed by the shortcut word you created.
+By typing `/lead` in the chat window, an operator can create a *Lead* in the *CRM* application.
+After typing `/lead`, create a title for this new lead, then press `Enter`.
 
-.. image:: responses/canned_response.png
+.. image:: responses/responses-lead.png
    :align: center
-   :alt: View of a chat window and the use of a canned response in Odoo Live Chat
+   :alt: View of the results from a /lead command in a Live Chat conversation.
+
+.. important::
+   The `/lead` command can only be used if the *CRM* app has been installed.
+
+To access the lead, click the link in the chat window, or navigate to the :menuselection:`CRM app`
+to view the :guilabel:`Pipeline`.
+
+The transcript of the conversation is added to the :guilabel:`Internal Notes` tab of the lead form.
+On the :guilabel:`Extra Information` tab of the lead form, the :guilabel:`Source` will be listed as
+:guilabel:`Livechat`.
+
+Leave
+-----
+
+If an operator types `/leave` in the chat window, they can automatically exit the conversation.
 
 .. seealso::
-   - :doc:`ratings`
+   - :doc:`/applications/sales/crm/acquire_leads`
+   - :doc:`/applications/services/helpdesk/overview/getting_started`
+
+Canned responses
+================
+
+*Canned responses* are customizable inputs where a *shortcut* stands in for a longer response. An
+operator will enter the shortcut, and it will automatically be replaced by the expanded
+*substitution* response in the conversation.
+
+Create canned responses
+-----------------------
+
+To create a new canned response, go to :menuselection:`Live Chat app --> Configuration --> Canned
+Responses --> New`.
+
+From here, choose a :guilabel:`Shortcut` and enter it into the :guilabel:`Shortcut` field.
+
+Then, click into the :guilabel:`Substitution` field, and enter the custom message that will be sent
+to visitors in place of the :guilabel:`Shortcut`. Click :guilabel:`Save`.
+
+.. tip::
+   Try to connect the :guilabel:`Shortcut` to the topic of the :guilabel:`Substitution`. The easier
+   it is for the *operators* to remember, the easier it will be to use the *canned responses* in
+   conversations.
```

Comment:
Gui use– use gui when referring to only the active element on the screen (the field), otherwise you're referring to the _shortcut_/_substitution_ the user is inputting.

Also no need for italics on operator or canned responses
```suggestion
From here, choose a shortcut and enter it into the :guilabel:`Shortcut` field.

Then, click into the :guilabel:`Substitution` field, and enter the custom message that will be sent
to visitors in place of the shortcut. Click :guilabel:`Save`.

.. tip::
   Try to connect the shortcut to the topic of the substitution. The easier it is for the operators
   to remember, the easier it will be to use the canned responses in conversations.
```